using-kubecost/navigating-the-kubecost-ui/teams.md

```diff
@@ -23,79 +22,254 @@ persistentVolume:
+         description: helm configured role
+         pages:
+           showOverview: true
+           showAllocation: true
+           showAsset: true
+           showCloudCost: true
+           showClusters: true
+           showExternalCosts: true
+           showNetwork: true
+           showCollections: true
+           showReports: true
+           showInsights: true
+           showActions: true
+           showAlerts: true
+           showBudgets: true
+           showAnomalies: true
+           showEfficiency: true
+           showSettings: true
+         permissions: admin
+         routes: []
+         allocationFilters:
+         - key: cluster
+           operator: ":"
+           value: cluster-one
+         assetFilters: []
+         cloudCostFilters: [] 
+       claims:
+         NameID: email@domain.com
+```
+
+Configure at least one team under `teamsConfig` with an associated role under the team's `roles`.
+
+Alternatively, a ConfigMap can be created manually, with `teamsConfigMapName` set to this ConfigMap's name.
+
+```yaml
+apiVersion: v1
+data:
+  rbac-teams-configs.json: '[json teams data]'
+kind: ConfigMap
+metadata:
+  name: <your-configmap-name>
+  namespace: <your-kc-namespace>
+```
+
+Note that setting a manually-configured ConfigMap name will override `teamsConfig`.
+
+For more information about teams and roles, see the [Adding a team](teams.md#adding-a-team) section below.
+
+## Claims
+
+Claims are part of the Identity Provider (IdP) response that provide information about the authenticated user. For SAML, these are relayed as part of the SAML response. For OIDC, these are returned as part of the access or ID tokens.
+
+Kubecost Teams uses these claims to map users to teams. As long as a group or attribute defined in your IdP is returned as part of the authorization process in the SAML response/OIDC token(s), it can be used in teams.
+
+For example, in OIDC, an ID token may contain a certain number of claims.
+
+```json
+{
+  "sub": "11111111",
+  "name": "John Example",
+  "email": "john.example@company.com",
+  "iss": "https:/authserver.com",
+  "groups": {
+    "kubecost_admins",
+  }
+  "iat": 1735722061,
+  "exp": 1735722062,
+}
+```
+
+While some of these claims (e.g. `iss`, `iat`, `exp`) are set automatically by the IdP and are unsuitable for matching, others (e.g. `groups`, `name`, `email`) can be used to map a Kubecost team to the user.
+
 ## Adding a team
 
-To create a team, select *Add Team*. The 'New Team' slide panel opens. Provide a name for your team, then choose the role (supports *Admin*, *Editor*, and *Read Only*). If this is your first team, select *Admin*.
+After initial configuration, more teams and roles can be added through the Teams page in the UI.
+
+### Teams and roles
+
+A team matches user claims from your configured IdP to a set of roles. Each role is essentially a set of permissions, such as over access level (admin/readonly/editor), page enablement states, filters, and so on.

```

Comment:
```suggestion
A team matches user claims from your configured IdP to a set of roles. Each role is essentially a set of permissions, such as over access level (admin/read-only/editor), page enablement states, filters, and so on.
```

What does the "over access level" phrase mean?

Reply:
A typo, I think, from when this said "is essentially a union of permissions", fixed the wording here in addition to changing readonly to read-only.